raiden-dapp/src/store/notifications/getters.ts

```diff
@@ -1,17 +1,30 @@
 import { GetterTree } from 'vuex';
 import { RootState } from '@/types';
-import { Notification, NotificationsState } from '@/store/notifications/types';
+import {
+  NotificationPayload,
+  NotificationsState,
+} from '@/store/notifications/types';
+import { NotificationImportance } from '@/store/notifications/notification-importance';
 
 export const getters: GetterTree<NotificationsState, RootState> = {
-  notifications: (state: NotificationsState): Notification[] => {
+  notifications: (state: NotificationsState): NotificationPayload[] => {
     return state.notifications.sort((a, b) => b.id - a.id);
   },
   nextNotificationId: ({ notifications }: NotificationsState): number => {
     if (notifications.length === 0) {
       return 1;
     }
-    return notifications
-      .map((notification) => notification.id)
-      .sort((a, b) => b - a)[0];
+    return (
+      notifications
+        .map((notification) => notification.id)
+        .sort((a, b) => b - a)[0] + 1
+    );
+  },
+  notificationQueue: (state: NotificationsState): NotificationPayload[] => {
+    return state.notifications.filter(
+      (notification) =>
+        notification.importance === NotificationImportance.HIGH &&
```

Comment:
So, only high-importance notifications go to the queue, right? Low notifications are just showno int eh snackbar?

Reply:
Excactly, I am not sure we have any low importance notification at the moment. But at least there is the mechanism to have notifications that don't show unless you go to the notification area.